src/main/java/com/greghaskins/spectrum/Spectrum.java

```diff
@@ -14,7 +14,9 @@
   /**
    * A generic code block with a {@link #run()} method to perform any action. Usually defined by a
    * lambda function.
+   * @deprecated since 1.0.1 - use {@link com.greghaskins.spectrum.Block} instead
    */
+  @Deprecated
```

Comment:
Then it stays - it's essentially just a rename of the public `Block` now. It kinda messes up the rest of the usage of Block in Spectrum, but better that "core" Block be the way forward.